<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ucDevelop</class>
 <widget class="QMainWindow" name="ucDevelop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ucDevelop</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ucdevelop.png</normaloff>ucdevelop.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionNewProject"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseFile"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Proje&amp;ct</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="separator"/>
    <addaction name="actionBuild"/>
    <addaction name="actionUpload"/>
    <addaction name="actionBuildAndUpload"/>
    <addaction name="separator"/>
    <addaction name="actionCleanProject"/>
    <addaction name="actionRebuild"/>
    <addaction name="separator"/>
    <addaction name="actionAddFile"/>
    <addaction name="actionAddClass"/>
    <addaction name="actionImportFolder"/>
    <addaction name="actionImportFile"/>
    <addaction name="separator"/>
    <addaction name="actionProjectSettings"/>
    <addaction name="separator"/>
    <addaction name="actionCloseProject"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewProject"/>
    <addaction name="actionViewFile"/>
    <addaction name="actionViewOutput"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProject"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionCloseFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="projectDock">
   <property name="windowTitle">
    <string>Pro&amp;ject</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="projectDockWidget">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="projectView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="fileDock">
   <property name="windowTitle">
    <string>&amp;File</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="fileDockWidget">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="fileWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outputDock">
   <property name="windowTitle">
    <string>O&amp;utput</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="outputDockWidget">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="clearOutputButton">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Clear output</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="KLed" name="outputLed">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="color">
        <color>
         <red>138</red>
         <green>138</green>
         <blue>138</blue>
        </color>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTextEdit" name="output">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
  </widget>
  <widget class="QToolBar" name="projectToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBuild"/>
   <addaction name="actionUpload"/>
   <addaction name="actionBuildAndUpload"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;as ...</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Sa&amp;ve Project</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Preference</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>C&amp;ut</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string> N&amp;ew Project</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open P&amp;roject</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Build</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Upload</string>
   </property>
  </action>
  <action name="actionBuildAndUpload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Build &amp;and Upload</string>
   </property>
  </action>
  <action name="actionProjectSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Project preference</string>
   </property>
  </action>
  <action name="actionViewProject">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Project</string>
   </property>
  </action>
  <action name="actionViewFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;File</string>
   </property>
  </action>
  <action name="actionViewOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Output</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="text">
    <string>Close &amp;file</string>
   </property>
  </action>
  <action name="actionCleanProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Clean project</string>
   </property>
  </action>
  <action name="actionRebuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rebu&amp;ild</string>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add &amp;file</string>
   </property>
  </action>
  <action name="actionImportFolder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>I&amp;mport folder</string>
   </property>
  </action>
  <action name="actionAddClass">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add c&amp;lass</string>
   </property>
  </action>
  <action name="actionImportFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Imp&amp;ort file</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close pro&amp;ject</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
